frontend/test/specs/post/timelineDirectiveSpec.js

```diff
@@ -124,33 +167,47 @@
             };
 
             rootScope.$emit("DELETED_POST", post_not_timeline);
-            expect(timelineCtrl.posts.length).toEqual(2);
-            expect(timelineCtrl.posts).toEqual(posts);
+            expect(timelineCtrl.posts.size()).toEqual(2);
+            expect(timelineCtrl.posts.data).toEqual(posts);
         });
 
         it('should not delete post because it has activity.', function() {
```

Comment:
this should be a test of Posts Factory

Reply:
yep, this test was there before the postsFactory